src/components/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import styled from 'styled-components'
+import Container from '../layout/Container'
+
+const Outer = styled.div<{ background?: string }>`
+  background: ${({ theme, background }) => (!background ? theme.colors.gradients.bubblegum : background)};
+`
+
+const Inner = styled(Container)`
+  padding-top: 32px;
+  padding-bottom: 32px;
+`
```

Comment:
`<Box />`?

Reply:
Hey I presumed this was targeted at the `Outer` 'div' on ln 6 Hachi? Changed if so!